types/chargebee__chargebee-js-react-wrapper/index.d.ts

```diff
@@ -0,0 +1,137 @@
+// Type definitions for chargebee-js-wrappers 0.2
+// Project: https://www.npmjs.com/package/@chargebee/chargebee-js-react-wrapper
+// Definitions by: Sagar Jain <https://github.com/sagar7993>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Component, CSSProperties } from 'react';
+
+export interface TokenizeAdditionalData {
+	readonly firstName?: string;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/chargebee__chargebee-js-react-wrapper/index.d.ts;
  Line: 9;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.